### ISSUE TITLE
MIgrating to Enhanced Recurring Donation

### ISSUE THREAD

**Issue by Billflorio** [SFDO-Community-Sprints/NPSP-Videography#53]
https://powerofus.force.com/s/article/NPSP-Enhanced-RD-Upgrade-Guide

It's a 55pp PDF right now!

**Comment by barbarachristensen** [comment]
Hi @pkasolves  or @andreahanson - I just requested access to this google doc so I can move it along the process.  Thanks!

**Comment by andreahanson** [comment]
Hello Barbara,

Access has been granted.
Andrea

Andrea Hanson
Senior Nonprofit Salesforce Consultant

Redpath Consulting Group

simplify | accelerate | transform

*Please note: All Redpath team members will be participating in our annual
business planning meeting and will have limited availability on January
19-20.*


phone: 612-314-6490

web: redpathcg.com

<https://www.facebook.com/redpathconsulting>
<https://twitter.com/redpathgroup>
<https://www.linkedin.com/company/redpath-consulting-group/>

<< Ask me about Enhancement Manager for Salesforce, your free CRM roadmap
app >>
<https://tracking.cirrusinsight.com/c1bc2610-cdd6-4085-ae27-e4628098b9f5/redpathcg-com-download-app-enhancement-manager-salesforce1>


This message and any attachments are solely for the intended recipient and
may contain confidential or privileged information. If you are not the
intended recipient, any disclosure, copying, use, or distribution of the
information included in this message and any attachments is prohibited. If
you have received this communication in error, please notify us by reply
email and immediately and permanently delete this message and any
attachments. Thank you.

**Comment by barbarachristensen** [comment]
thanks!

**Comment by Billflorio** [comment]
@pkasolves Can you review one more time for me?

**Comment by barbarachristensen** [comment]
Hey @Billflorio THis one took some pretty decent re-write and slimming down, but it's still pretty long and maybe you won't think some of it is film-able? Have a look. If it looks like you can get screen steps to go with that work for you, then @coriobriensfdo can review it. It's really just an overview of the process, not all the steps, of course.

**Comment by Billflorio** [comment]
I think it's pretty good. The hard part is actually capturing the steps, you sort of only get one chance to upgrade

**Comment by barbarachristensen** [comment]
Yeah! :)  Ok @coriobriensfdo  this one is for you to approve, more coming next day or so!

**Comment by coriobriensfdo** [comment]
@barbarachristensen I had to request access to the draft script.

**Comment by coriobriensfdo** [comment]
I got access and reviewed! :)
